packages/Uploader/src/components/DropZone/DropZone.js

```diff
@@ -18,13 +26,14 @@ export default function DropZone() {
     <React.Fragment>
       {I18n.t("uploader.drop_files_here_or")}&nbsp;
       <Button
+        aria-label={I18n.t("uploader.choose_from_computer_a11y")}
         data-pka-anchor="uploader-dropZone-link"
         kind={Button.types.kind.LINK}
         onClick={() => {
           refInput.current.click();
         }}
         isSemantic={false}
-        tabIndex={-1}
+        tabIndex={0}
```

Comment:
Similar here, Button component should be focusable by default

Reply:
Sorry I think I didn't submit my last comment, because you changed the `tabIndex` here, there'll be 2 elements focusable on the screen, one is this button, the other one is the below, is this what we expected?

Reply:
Hmm, yeah I wasn't sure about that-- right now, there's two focusable "links" to select files from computer. I could disable the one coming from line 53?

Reply:
fyi here might be a solution if we only want 1 focusable element https://codesandbox.io/s/affectionate-shape-551f7?file=/src/App.js although it might be ugly